Foundation/Thread.swift

```diff
@@ -183,6 +229,10 @@ open class Thread : NSObject {
         } else {
             _thread = nil
         }
+#elseif os(Windows)
+        _thread = self.withRetainedReference {
+            return _CFThreadCreate(_attr, NSThreadStart, $0)
+        }
```

Comment:
Isnt this block redundant as it is a duplicate of the block below?

Reply:
Not when it was originally written :-).